Gemfile

```diff
@@ -23,6 +23,9 @@ gem 'mysql2'
 # for observing records
 gem 'rails-observers'
 
+# for rating
+gem 'ratyrate', github: 'wazery/ratyrate', branch: 'master'
```

Comment:
Does using the gem from the **master** branch instead of the released version something we can agree upon?